### ISSUE TITLE
Add all {action} to a single request in count_history

### ISSUE THREAD

**Issue by tristen** [osmlab/to-fix-backend#23]
Discussed in chat with @aaronlidman. For the [endpoint documented here](https://github.com/osmlab/to-fix-backend/blob/eca1ca32f2811d36f2e4e2ba50430e290f84a0e7/API.md#get-count_historytaskactiongrouping), ideally all actions are grouped in a single request rather than split out into many. Otherwise needless multiple requests are made to fetch more action data. Something like:

``` json
{
    "start": 1426809600,
    "skipped": 12,
    "edited": 17,
    "fixed": 22
}
```

would be ideal.

**Comment by aaronlidman** [comment]
@tristen this is live, for example: http://54.147.184.23:8000/count_history/unconnectedmajor/day

**Comment by aaronlidman** [comment]
Ugg one sec, unixtime is a string, need to redeploy. 2 minutes please.

**Comment by aaronlidman** [comment]
Fixed in 3be5af308fb2977f8d754dfdb1f459cd08270c4d
